internal/config/main_config.go

```diff
@@ -64,6 +64,9 @@ var config struct {
 		ClientID       string
 		ClientSecret   string
 	}
+	Gcp struct {
```

Comment:
We do have `AWS` in the main config, so let's keep this for this case:

```suggestion
	GCP struct {
```

Reply:
for some reason that didn't work for @adiabramovitch, but might be issue somewhere else? If it works, I'd also prefered to be consistent.